openassessment/xblock/grade_mixin.py

```diff
@@ -350,10 +357,11 @@ def _get_assessment_part(title, part_criterion_name, assessment):
         else:
             peer_assessment_part = None
         example_based_assessment_part = _get_assessment_part(
-            _('Example-Based Grade'), criterion_name, example_based_assessment
+            _('Example-Based Grade'), _('Example-Based Comments'), criterion_name, example_based_assessment
         )
         self_assessment_part = _get_assessment_part(
             _('Self Assessment Grade') if is_staff else _('Your Self Assessment'),
+            _('Your Comments'),
```

Comment:
Nit: can you please add a comment stating that this new value is only used on the LMS view? I was worried because the line above made me think that we needed a different value when is_staff is True, but we don't because this new field is not used in the learner info part of the staff tools.